lib/Sema/CSStep.cpp

```diff
@@ -346,8 +346,46 @@ StepResult ComponentStep::take(bool prevFailed) {
 
   /// Try to figure out what this step is going to be,
   /// after the scope has been established.
+  std::vector<BindingSet> currentBindings;
+  auto bestBindings = CS.determineBestBindings([&](const BindingSet &bindings) {
```

Comment:
No `Potential Bindings:` or `Initial state:` header?

Reply:
The header (prev called `Current Bindings`, but I've updated it to `Potential Bindings` now) is printed after I collect/check if `bestBindings` is empty. Otherwise, the header will print even when there are no bindings.

Reply:
Ah, I see!